### DISCUSSION TITLE
Set the app  to a language different from the mobile one

### DISCUSSION THREAD

**Discussion by deltazerorsan** [owntracks/android#1169] [Q&A]
First off all, thanks.

I have mi mobile in my own language, Spanish, but I prefer to have the OwnTrack app in English. In this way I can follow the booklet easily.

Is there a way to do that?

Rom

**Comment by growse** [comment]
My initial reaction to this was "this is an entirely sensible thing to want". But then I did a little digging and it turns out that Android doesn't make this easy to do.

There's existing apps that do this already (https://www.androidheadlines.com/2021/01/google-maps-different-app-language.html), and the press reports that Google are looking at supporting this in general on the device (https://www.androidcentral.com/google-per-app-language-settings-android). 

In theory it's possible for an app to override the device settings, but it looks pretty hacky. I'd be inclined to wait until Android lets you do this properly, rather than try and support it somehow in-app.
